ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -137,10 +116,29 @@ namespace OrdinalAxis {
      *
      * */
 
+    export declare class Composition extends Axis {
+        forceOrdinal?: boolean;
+        isInternal?: boolean;
+        ordinal: OrdinalAxis.Additions;
```

Comment:
Inside the namespace it is not necessary to use reference the own namespace. So OrdinalAxis.Additions becomes Additions, and OrdinalAxis.Composition becomes Composition.